README.md

```diff
@@ -142,6 +142,29 @@ Response onRequest(RequestContext context) {
 }
 ```
 
+We can also return any Dart object in the `body` of the  `Response.json` constructor as long as it has a `toJson` and `fromJson` function (for example, using [json_serializable](https://pub.dev/packages/json_serializable)):
+
+```dart
+import 'package:dart_frog/dart_frog.dart';
+
+Response onRequest(RequestContext context) {
+  return Response.json(
+    body: MyCustomResponsePayload('hello world!'),
+  );
+}
+
+@JsonSerializable()
+class MyCustomResponsePayload {
+  const MyCustomResponsePayload(this.responseParam);
+  
+  factory MyCustomResponsePayload.fromJson(Map<String, dynamic> json) => _$MyCustomResponsePayloadFromJson(json);
+  
+  Map<String, dynamic> toJson() => _$MyCustomResponsePayloadToJson(this);
+  
+  final String responseParam;
+}
+```
```

Comment:
```suggestion
```dart
import 'package:dart_frog/dart_frog.dart';

Response onRequest(RequestContext context) {
  return Response.json(
    body: User(name: 'Dash', age: 42),
  );
}
```

```dart
import 'package:json_annotation/json_annotation.dart';

part 'user.g.dart';

@JsonSerializable()
class User {
  const User({required this.name, this.age});
  
  final String name;
  final int age;
  
  Map<String, dynamic> toJson() => _$UserToJson(this);
}
```
```